.github/workflows/ci.yaml

```diff
@@ -18,19 +18,19 @@ jobs:
     name: Lint
     uses: canonical/data-platform-workflows/.github/workflows/lint.yaml@v13.2.0
 
-  unit-test:
-    name: Unit test charm
-    runs-on: ubuntu-latest
-    timeout-minutes: 10
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Install tox & poetry
-        run: |
-          pipx install tox
-          pipx install poetry
-      - name: Run tests
-        run: tox run -e unit
```

Comment:
Temp.: Will revert it once the main integration tests pass and we are sure of the changes below (there are a lot of unit tests that will need to be updated here)